pyproject.toml

```diff
@@ -16,6 +16,8 @@ backoff = "^2.2.1"
 boto3 = "^1.34.45"
 moto = "^5.0.3"
```

Comment:
Remove `moto` from [tool.poetry.dependencies] and update the version to "^5.0.3" in [tool.poetry.group.dev.dependencies]